lib/src/api/observable.dart

```diff
@@ -47,15 +49,18 @@ ObservableValue<T> observable<T>(T initialValue, {String name}) =>
 /// A computed value is _cached_ and it recomputes only when the dependent observables actually
 /// change. This makes them fast and you are free to use them throughout your application. Internally
 /// MobX uses a 2-phase change propagation that ensures no unnecessary computations are performed.
-ComputedValue<T> computed<T>(T Function() fn, {String name}) =>
-    ComputedValue(fn, name: name);
+ComputedValue<T> computed<T>(T Function() fn,
+        {String name, ReactiveContext context}) =>
+    ComputedValue(context ?? mobxContext, fn, name: name);
 
 /// Creates a simple Atom for tracking its usage in a reactive context. This is useful when
 /// you don't need the value but instead a way of knowing when it becomes active and inactive
 /// in a reaction.
 ///
 /// Use the [onObserved] and [onUnobserved] handlers to know when the atom is active and inactive
 /// respectively. Use a debug [name] to identify easily.
-Atom createAtom({String name, Function onObserved, Function onUnobserved}) =>
-    Atom(name ?? 'Atom@${ctx.nextId}',
-        onObserve: onObserved, onUnobserve: onUnobserved);
+Atom createAtom({String name, Function onObserved, Function onUnobserved}) {
```

Comment:
No context argument here? Any reason why we are picking `mobxContext` directly?

Reply:
Also the context should be a named argument

Reply:
Yeah, missed this one.